src/widgets/WalletModal/icons/Brave.tsx

```diff
@@ -1,49 +1,97 @@
 import React from "react";
-import LegacySvg from "../../../components/Svg/LegacySvg";
 import { LegacySvgProps } from "../../../components/Svg/types";
 
 const Icon: React.FC<LegacySvgProps> = (props) => {
```

Comment:
Could you move these wallet icons into the SVG folder and remove the LegacySvgProps